### PR TITLE
Closes #72

### LINKED ISSUE #72: Create a Database
Desc: Create a new database
DOD: Database is able to store submitted photos, the users, photo names, and tags.

### PR DESCRIPTION
**Pull request by AlainNshimirimana**
After many tries and errors, I finally managed to create a database and an initial migration.
I put everything under the "database-feature-Alain" branch.
Please take a look at it and let me know what needs to be changed

### CODE REVIEW

File: photo_points/Startup.cs [hunk 1]
```diff
@@ -39,6 +39,13 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IAdminReviewServices, AdminReviewServices>();
             services.AddTransient<IAdminReviewRepository, FakeAdminReviewRepository>();
 
+            //add services for the PhotoDataContext
+            services.AddDbContext<Models.PhotoDataContext>(options =>
+            {
+                options.UseSqlServer(Configuration.GetConnectionString("PhotoPointSubmissions"));
+                //options.UseSqlite(Configuration.GetConnectionString("IndyBooks-Mac-Sqlite"));
```
Comment (floydfix) [inline photo_points/Startup.cs]: And this

File: photo_points/Models/Tag.cs [hunk 2]
```diff
@@ -13,7 +13,8 @@ public class Tag
         [Required(ErrorMessage = "Tag name is required.")]
         public string tagName { get; set; }
 
-        public ICollection<User> users { get; set; }
-        public ICollection<Capture> captures { get; set; }
+        public User user { get; set; }
+        //public ICollection<Capture> captures { get; set; }
```
Comment (floydfix) [inline photo_points/Models/Tag.cs]: You should delete this commented code

**Comment by NathanaelRorberg** [comment]
I can see the migrations but I can't seem to find the database in the sql server object explorer window. What's the name of the database?

**Review by NathanaelRorberg — CHANGES_REQUESTED** [review]
I can see the migrations but I can't seem to find the database in the sql server object explorer window. What's the name of the database?

**Comment by AlainNshimirimana** [comment]
Nathan, the database is called AdminService. If you can't find it, try the following.
1. dotnet ef database drop
2. dotnet ef database update
3. Build solution and run the app
4. Check sql server explorer again

**Review by floydfix — COMMENTED** [review]
Also, un-comment line 16 in PhotoDataContext.cs and the db will create the first time run.
Meant to request these as changes.

File: photo_points/Models/Tag.cs (hunk 2 above)
Comment (AlainNshimirimana) [reply photo_points/Models/Tag.cs]: Yep. Done.

**Review by floydfix — APPROVED** [review]
Sorry, I guess you need to have some more code that calls this PhotoDataContext class to get that to auto build the Database. I am not sure it is totally needed at this point since the next bit we do with the db should be to hook it  up and that will call the ensureCreated code.

**Review by taerichard — COMMENTED** [review]
I was told it's best practice to capitalize the first letter of public fields on a class.  And lowercase first letter on private fields.

**Review by taerichard — APPROVED** [review]

**Review by gjesionowski — APPROVED** [review]

**Review by NathanaelRorberg — APPROVED** [review]

**Pull request closed without merging**